modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/pages/questions/NewQuestion.es.js

```diff
@@ -200,6 +198,8 @@ export default withRouter(
 									value={headline}
 								/>
 
+								<span className="d-flex justify-content-end mt-2 text-secondary">{`${headline.length} / 75`}</span>
```

Comment:
**75** is a magic number, and shouldn't be placed here.

You can create a `const` for example `const HEADLINE_MAX_LENGTH = 75` and reuse it where **75** is placed, the same for the <input in this file

The const can be placed before `withRouter`

Reply:
<img width="1109" alt="image" src="https://user-images.githubusercontent.com/22279592/188703159-380d0d59-6953-43be-b334-bdbb1c4be6d4.png">

Also limit count is alone in one line, and the description in another. Would be possible to place both texts in the same line?

Reply:
@Claudionilton